evadb/parser/evadb.lark

```diff
@@ -172,11 +174,19 @@ explain_statement: EXPLAIN explainable_statement
 
 explainable_statement : select_statement | insert_statement | update_statement | delete_statement | create_table
 
+// Context Statements
+
+use_statement: USE database_name "{" query_string "}" // One shortcoming that query string cannot have parenthesis
```

Comment:
Is this the limitation of the lark or ? Why we can not have parenthesis?

Reply:
I will remove this comment. The native query can have parenthesis after I change to use curly bracket if that makes sense
```sql
// now
USE postgres {
   // some query
}

// before
USE postgres (
    // some query
)
```

If I use the parenthesis and the native query has arbitrary parentheses as well, I have some trouble of coming up with a grammar that works for all cases.

Reply:
Let me take care of it